crates/services/relayer/src/config.rs

```diff
@@ -45,7 +45,7 @@ pub struct Config {
 
 #[allow(missing_docs)]
 impl Config {
-    pub const DEFAULT_LOG_PAGE_SIZE: u64 = 5;
+    pub const DEFAULT_LOG_PAGE_SIZE: u64 = 10_000;
```

Comment:
This is a pretty large increase, couldn't this worsen our rate limiting issues?

Reply:
It should decrease the number of requests=) I used `100_000` locally during synchronization with the network and it was only several requests
<img width="601" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/7de193c5-c017-44c7-a8ac-14edc107a41c">